sites/partners/pages/listings/[id]/applications/pending/index.tsx

```diff
@@ -0,0 +1,187 @@
+import React from "react"
```

Comment:
This file and the resolved file are almost identical, can we extract the shared page content into a shared component in partners?